### PR TITLE
Prepare 0.10.0 Release

### PR DESCRIPTION
**Pull request by ciyer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+* consider falsy string values for maintenance page (#903) 7c6bb8c, closes #903
+* improve styling of notebooks (#870) edc6c6d, closes #870
+* remove conditional variables from template file (#910) 927a6d3, closes #910
+* update the jupyter button used in file view (#909) c359034, closes #909
+
+
+### BREAKING CHANGES
+
+**environments**
+* Requires backend components supporting anonymous users environments
```
Comment (ableuler) [inline CHANGELOG.md:36]:
```suggestion
* Requires backend components supporting anonymous users environments ([renku-notebooks 0.7.2](https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/0.7.2) or later)
```

**Comment by rokroskar** [comment]
Could you change the link to the tutorial so that it works with https://github.com/SwissDataScienceCenter/renku/pull/991 and so we can merge that PR as well?

**Comment by ciyer** [comment]
> Could you change the link to the tutorial so that it works with [SwissDataScienceCenter/renku#991](https://github.com/SwissDataScienceCenter/renku/pull/991) and we can merge that PR as well?

This is already in:
`* update links to new renku tutorial url (#911) 80bf4f3, closes #911 #892`

**Comment by rokroskar** [comment]
oh oops, missed that one. Thanks!

**Comment by vfried** [comment]
The dataset_name to small_name change is missing, but i guess it's not important to put that here.

**Comment by ciyer** [comment]
> The dataset_name to small_name change is missing, but i guess it's not important to put that here.

Good catch! Not sure how that got missed.

**Comment by ciyer** [comment]
> Ooops, I didn't notice this was marked as ready to review.
> 
> Would it make sense to add the full URL to issues and PRs in the changelog, as in the previous releases? Otherwise, links don't work everywhere.

Oops! I tested the changelog in GitHub, so I missed this because the links worked there.

**Pull request merged**